### PR TITLE
Edit UI: Unit Test Backfill 

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#4984

*Description of changes:*

* Implemented Unit testing for various correction components

* Tweaked YourCompany correction v-chip

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Company/CompanyName/CorrectNameOptions.vue [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <v-expansion-panels v-model="panel">
       <v-expansion-panel
         v-for="(item,i) in displayedOptions"
+        :id="item.id"
```
Comment (severinbeauvais) [inline src/components/Company/CompanyName/CorrectNameOptions.vue:9]: Maybe add a name to this panel's id?

Comment (cameron-freshworks) [reply src/components/Company/CompanyName/CorrectNameOptions.vue:9]: For sure.
Something like `:id="x-panel: ${item.id}" ? Is that what you mean?

File: src/components/DefineCompany/YourCompany.vue [hunk 2]
```diff
@@ -15,10 +15,10 @@
                 v-if="companyNameChanges"
                 class="ma-2 pa-2"
                 color="primary"
-                pill
+                label
                 x-small
               >
-                Corrected
+                CORRECTED
```
Comment (severinbeauvais) [inline src/components/DefineCompany/YourCompany.vue:21]: Lekshmi's base class addition should handle this (but doesn't hurt either way).

File: src/components/DefineCompany/YourCompany.vue [hunk 3]
```diff
@@ -15,10 +15,10 @@
                 v-if="companyNameChanges"
                 class="ma-2 pa-2"
                 color="primary"
```
Comment (severinbeauvais) [inline src/components/DefineCompany/YourCompany.vue:17]: Color should be "#1669BB". Watch for the merge with Lekshmi's PR so you don't undo her change.

**Review by severinbeauvais — APPROVED** [review]

File: src/components/Company/CompanyName/CorrectNameOptions.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/Company/CompanyName/CorrectNameOptions.vue:9]: Yes.

File: src/components/DefineCompany/YourCompany.vue (hunk 3 above)
Comment (cameron-freshworks) [reply src/components/DefineCompany/YourCompany.vue:17]: kk, will do!

**Review by vysakh-menon-aot — APPROVED** [review]

File: src/components/DefineCompany/YourCompany.vue [hunk 4]
```diff
@@ -14,8 +14,8 @@
               <v-chip
                 v-if="companyNameChanges"
                 class="ma-2 pa-2"
-                color="primary"
-                pill
+                color="#1669BB"
```
Comment (severinbeauvais) [inline src/components/DefineCompany/YourCompany.vue:17]: If you set a named color (not "primary") then you have to set the text-color property as well ("white").

Comment (cameron-freshworks) [reply src/components/DefineCompany/YourCompany.vue:17]: ah. i see! Fixed in next commit

**Pull request merged**